src/jit/flowgraph.cpp

```diff
@@ -5763,7 +5772,7 @@ void Compiler::fgFindBasicBlocks()
             impInlineInfo->iciCall->gtCallMoreFlags |= GTF_CALL_M_DOES_NOT_RETURN;
```

Comment:
Right, earlier detection is a good idea, since we'd rather fail with the no return reason than some other reason. I am just asking if it means the later detection becomes redundant (I suspect it does since not much happens in between).